resources/charts/dataspace-connector/values.yaml

```diff
@@ -27,25 +27,29 @@ securityContext: {}
   # runAsNonRoot: true
   # runAsUser: 1000
 
+ports: {}
+
+volumes: []
+  # For a comprehensive overview of supported volume types, please refer to: https://kubernetes.io/docs/concepts/storage/volumes/
+  # Below example will create a volume from the content of the `log-config` ConfigMap keyed with `log_level`:
+  # - name: config-vol
+  #   configMap:
+  #     name: log-config
+  #     items:
+  #     - key: log_level
+  #       path: log_level
+
 edc:
-  port: 8181
-  path: /api
-  managementApi:
-    enabled: true
-    port: 9191
-    path: /api/v1/data
-  idsApi:
-    port: 8282
-    path: /api/v1/ids
   env: {}
   envSecrets: {}
+  volumeMounts: []
+  # Mount the provided volume in the POD filesystem at the specific path, for example:
+  # - name: config-vol
+  #   mountPath: /etc/config
+  # will mount the `config-vol` volume at /etc/config
```

Comment:
Why do we need to have `env`, `envSecrets`, `volumeMounts` under `edc`? Could be on the root level or not?

Reply:
We could, but for now all POD settings are placed under `edc:`. `volumeMounts` refers to the way a volume is mounted into the container so for me this hierarchy makes sense.